### ISSUE TITLE
[FEATURE] Linux aarch64 builds

### ISSUE THREAD

**Issue by priteau** [prometheus-msteams/prometheus-msteams#229]
Would it be possible to provide release binaries for Linux aarch64? Thanks.

**Comment by Knappek** [comment]
Yes sure. Are you interested to contribute that enhancement as you can likely better test it.
